### DISCUSSION TITLE
Can't run on centos7.9

### DISCUSSION THREAD

**Discussion by wongsimon** [dlemstra/Magick.NET#1239] [Help]
Unable to load shared library 'Magick.Native-Q8-x64.dll' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: libMagick.Native-Q8-x64.dll: cannot open shared object file: No such file or directory   at ImageMagick.Environment.NativeMethods.X64.Environment_SetEnv(IntPtr name, IntPtr value)
   at ImageMagick.Environment.NativeEnvironment.SetEnv(String name, String value) in /_/src/Magick.NET/Native/Helpers/Environment.cs:line 116

Code:
using (var image = new MagickImageCollection(new FileInfo(psd)))
{
      image.Write(psd.Replace(".psd", ".gif"),MagickFormat.Gif);
 }

Environment:Centos7.9 x64, .Net6, Magick.Net 12.1.0

![1661851988764](https://user-images.githubusercontent.com/6326997/187402799-9619f9a4-be0f-4d7e-a7f5-94390d9b4d63.png)

**Comment by dlemstra** [comment]
What is the output of `ldd Magick.Native-Q8-x64.dll.so`? I suspect you need at least CentOS 8/9.
